src/sys/output.cxx

```diff
@@ -153,4 +155,21 @@ ConditionalOutput output_error(Output::getInstance());
 ConditionalOutput output_verbose(Output::getInstance(), false);
 ConditionalOutput output(Output::getInstance());
 
+void backupfile(const char* filename) {
+  struct stat buffer;
+  if (stat(filename, &buffer) == 0){
+    char filename_[strlen(filename)+10];
+    int j=0;
+    do {
+      if (j > 1000000) {
```

Comment:
Magic number should be `constexpr int` and named -- `max_backup_files`?

Also this is a pretty big number, and quite possibly more than can be supported in a single directory, depending on the filesystem. I would be surprised if a program silently made a million backups for me! Maybe this should just be one or two?

Reply:
Changed to a warning with much lower number :+1: